### PR TITLE
[Card#35]-Delete calendar from local db and remote api.

### PR DESCRIPTION
**Pull request by HarshwardhanSingh**
Trello card: https://trello.com/c/QQ8Oaigx

Branch out from **master** branch

Rebased with **edit_calendars_locally_and_remote** branch

### CODE REVIEW

File: app/controllers/calendars_controller.rb [hunk 1]
```diff
@@ -46,8 +46,33 @@ def create
...
+    calendar.update(summary: params[:summary])
+    calendar = {summary: params[:summary]}
+    @client.update_calendar(params[:id], calendar)
+    redirect_to calendars_path
+  end
+
+  def destroy
+    @client.delete_calendar(params[:id])
+    calendar = Calendar.find_by(google_id: decode_id(params[:id]))
+    calendar.leaves.each do |leave|
```
Comment (deepak) [inline app/controllers/calendars_controller.rb]: try `destroy_all` http://apidock.com/rails/ActiveRecord/Relation/destroy_all

File: app/controllers/calendars_controller.rb [hunk 2]
```diff
@@ -46,8 +46,33 @@ def create
...
+      leave.destroy
+    end
+    invites = Invite.where(calendar_id: calendar.id)
+    invites.each do |invite|
```
Comment (deepak) [inline app/controllers/calendars_controller.rb]: same here. http://apidock.com/rails/ActiveRecord/Relation/destroy_all

**Comment by deepak** [comment]
@HarshwardhanSingh one comment about `delete_all` otherwise good to go

**Comment by HarshwardhanSingh** [comment]
@deepak 
calendars = Invite.where(user_id: current_user.id, is_default: true).select(:calendar_id) doesn't work.
calendars = Invite.where(user_id: current_user.id, is_default: true).map(&:calendar_id) works fine.
at line no 82 where we are rendering default calendars.

**Pull request closed without merging**